src/entities/SecurityToken/Permissions.ts

```diff
@@ -186,8 +178,6 @@ export class Permissions extends SubModule {
 
   /**
    * Returns the list of delegate addresses and details that hold a specific role
```

Comment:
Return the list ...

Reply:
Please apply this same criteria to the rest of the descriptions in the file (I can't comment beyond this one)